assets/sass/kerrokantasi/_commentlist.scss

```diff
@@ -96,6 +105,47 @@
               font-size: 1.4rem;
           }
       }
+
+    .hearing-comment-show-more {
+        border-top: 1px solid $light-gray;
+        font-weight: bold;
+        margin-top: $margin-default;
+        padding: $padding-default 0 0 0;
+        text-align: center;
+        &__wrapper {
+          border-bottom: 1px solid $black;
+          padding: 0 $padding-small;
+
+          &:hover {
+            border-bottom: 1px solid $hover-blue;
```

Comment:
This `$hover-blue` value needs to be defined in the `assets/sass/kerrokantasi/variables.scss` file as well with the `!default` flag. Without it being assigned the build will crash with white label styles applied.

A good value for this would be `$hover-blue: $theme-info;`

Reply:
removed, theme-info now being used instead